### ISSUE TITLE
Connect to Nextcloud: Login Server URL provision – source of URL unclear

### ISSUE THREAD

**Issue by jdittrich** [nextcloud/desktop#1160]
### User Story
As a new user, I want to set up the client with my nextcloud sync client app. 

### What I did 
1) I installed the sync app.
2) Use the  the "Connect to nextcloud"-dialog
3) Clicked "Login"

I am asked for a server address, but I do not know how to retrieve it. I was successful in copypasting the URL that I have when use when I am using the web version of my nextcloud.  

### Possible fix

Ease finding the "right" URL. In the most simple case, this could be a link to a page explaining how to find the needed URL. 

Another feature that I saw at [DavX5](https://www.davx5.com/) was choosing your provider from a list of possible providers (just like you do in the screen before under "Register…" which would supply the configured URL for this provider. 

Minor: 
* Also here, the [carousel is distracting](https://github.com/nextcloud/desktop/issues/1159)
* The connection security icon looks a bit too big 
![image](https://user-images.githubusercontent.com/3416487/54878728-fbb74700-4e30-11e9-9748-5775743f30ed.png)

### Client configuration
Client version: 2.5.1final (build 20181204)

Operating system:Ubuntu 18.04

Client package (From Nextcloud or distro) (Linux only): AppImage

**Comment by jdittrich** [comment]
ping @jancborchardt

**Comment by jancborchardt** [comment]
@jdittrich very good point, had the same one in a recent test. Like "Ok what is the 'Server address' … nextcloud.com?".

Maybe just some explanatory text below would be enough, like:

> This is the link to your Nextcloud web interface when you open it in the browser. It looks like `https://cloud.example.com` or `https://example.com/cloud`

What do you think @jdittrich @nextcloud/designers?





---------------


> Also here, the carousel is distracting

Yup. The carousel should only show in the first view, and go away if you click any of the buttons. Same with the other buttons to "Sign up with a provider" and "Host your own server". There should simply be a back button on the bottom left. @camilasan?

> The connection security icon looks a bit too big

Right, can we reduce the size a little bit @camilasan?

**Comment by jdittrich** [comment]
> …This is the link to your Nextcloud web interface when you open it in the browser. It looks like https://cloud.example.com or https://example.com/cloud…

Perfect, this would have helped me, particularly since it is connected to an activity ("when you open it in a browser"). 

I still find integrating the list of providers a neat idea, but this would have been a "delight". Not pondering about the address is a "basic need" (in [KANO](https://en.wikipedia.org/wiki/Kano_model)-terms)

> carousel should only show in the first view, and go away if you click any of the buttons

That makes sense. I do understand that one wants a carousel for marketing. Once users enter the process, there should only be focus on this process (and possibly short explanations to make sense of possible actions).

**Comment by jancborchardt** [comment]
>I still find integrating the list of providers a neat idea, but this would have been a "delight".

Right, basically like how mail apps often show this list with most popular providers like "Gmail, Outlook, …, Custom server" – do you mean like that? And we would list the providers which are part of the simple sign up.

cc @camilasan @rullzer what do you think? That’s a good point to pick people up when they signed up through the mobile app or website. Would also be good for the mobile apps in the "Log in" flow cc @tobiasKaminsky @AndyScherzinger @marinofaggiana

**Comment by jdittrich** [comment]
> show this list with most popular providers like "Gmail, Outlook, …, Custom server" – 
> do you mean like that?

yes!

**Comment by jancborchardt** [comment]
@misch7 this is something to keep in mind for the account dialog update, we could just add this explanatory text below the server address field as mentioned above:

> This is the link to your Nextcloud web interface when you open it in the browser. It looks like `https://cloud.example.com` or `https://example.com/cloud`
